Sources/Views/InputTextView.swift

```diff
@@ -232,25 +233,28 @@ open class InputTextView: UITextView {
     open override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
         
         if action == NSSelectorFromString("paste:") && UIPasteboard.general.image != nil {
-            return isImagePasteEnabled
+            if isImagePasteEnabled || (inputBarAccessoryView?.inputPlugins.count ?? 0) > 0 {
+                return true
+            }
         }
         return super.canPerformAction(action, withSender: sender)
     }
     
     open override func paste(_ sender: Any?) {
         
-        guard let image = UIPasteboard.general.image else {
-            return super.paste(sender)
-        }
-        if isImagePasteEnabled {
-            pasteImageInTextContainer(with: image)
-        } else {
-            for plugin in inputBarAccessoryView?.inputPlugins ?? [] {
-                if plugin.handleInput(of: image) {
-                    return
+        if let image = UIPasteboard.general.image {
+            if isImagePasteEnabled {
+                pasteImageInTextContainer(with: image)
+                return
+            } else {
+                for plugin in inputBarAccessoryView?.inputPlugins ?? [] {
+                    if plugin.handleInput(of: image) {
+                        return
+                    }
                 }
             }
         }
+        return super.paste(sender)
```

Comment:
Is the intention here to execute `return super.paste(sender)` if there are no plugins, or no plugins return `true` from `handleInput(of: image)`?